argo-cd-apps/base/ci-workspaces/ci-workspaces.yaml

```diff
@@ -0,0 +1,42 @@
+apiVersion: argoproj.io/v1alpha1
+kind: ApplicationSet
+metadata:
+  name: ci-workspaces
+spec:
+  generators:
+    - merge:
+        mergeKeys:
+          - nameNormalized
+        generators:
+          - clusters:
```

Comment:
you would need something like this for selecting only the production RH cluster https://github.com/redhat-appstudio/infra-deployments/blob/main/argo-cd-apps/base/tekton-ci/tekton-ci.yaml#L16